package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
I'd love to learn more about concerns on bumping the version. let's probably chat after the retro tomorrow. 

The challenge today is that when you bump the version in the coda repo, it comes with the latest released `sdkVersion` and all the latest non-released features. For example the `ImageSchema` item in the change log was not released with 1.0.2. However the packs-sdk used in the coda repo may already have this feature but is still tagged with 1.0.2. On the other hand, the external pack makers would never have the `ImageSchema` feature until 1.0.3. 

This leads us to a question: does the `ImageSchema` feature start from 1.0.2 or 1.0.3? since the behavior is different in the coda repo (i.e. pack runtime files) and in pack makers' world (i.e. pack build time).

Reply:
actually let me turn to a slack if this needs to be discussed in zoom.